### DISCUSSION TITLE
Container statuses set to WAIT_START for both Search Node and Forward Node

### DISCUSSION THREAD

**Discussion by jvossler** [Security-Onion-Solutions/securityonion#8017] [Unsupported Versions]
I am having an issue with getting my Search Node and Forward Node docker containers past WAIT_START. I ran both `salt-call pillar.get sensor:mainint` and `salt-call pillar.get host:mainint` and received the message:
"Attempt to authenticate with the salt master failed with timeout error"

This was after verifying the correct interface that is connected to the network through reviewing `/opt/so/saltstack/local/pillar/minions/<hostname>_sensor.sls`

IP address in `<hostname>_sensor.sls` is correct. Funny thing is they were all working yesterday (updates being pushed, `so-test` working perfectly), then I powered all three of my VMs down. When I powered them up again today, this issue popped up. The issue has persisted through resetting all three VMs back to known good snapshots.

**Comment by dougburks** [comment]
Is the salt master service running on the manager?

Can the nodes connect to the salt master ports?
